ExceptionsAndLogging/src/scit/homework/exceptions/Main.java

```diff
@@ -0,0 +1,28 @@
+package scit.homework.exceptions;
+
+import java.util.TreeSet;
+
+            public class Main {
+                public static void main(String[] args) {
+
+                    Student mara = new Student("Mara", "Amm", "05 02 2000",'F', "8656565656" );
+                    Student mary = new Student("Mary", "Abbb", "05 02 1999",'f', "556565677" );
+                    Student mario = new Student("Mario", "Far", "05 07 2000",'M', "923432656" );
+
+
+                    TreeSet<Student> studentsSet = new TreeSet<>(new cnpComparator());
```

Comment:
Lista de studenti ar trebui sa fie intr-o alta clasa,  (de ex StudentRepository). In aceasta clasa ar trebui sa existe cele 4 metode (add student, delete student (by ID),,retrieve all students with Age X, list students order by Last Name or Birth Date)